packages/core/src/rest/ZosmfRestClient.ts

```diff
@@ -21,6 +21,13 @@ import { ZosmfHeaders } from "./ZosmfHeaders";
  */
 export class ZosmfRestClient extends RestClient {
 
+    /**
+     * Construct zosmf rest client to use content decoding
+     * @param mSession The session object to use
+     */
+    constructor(mSession: AbstractSession){
+        super(mSession);
+    }
```

Comment:
This constructor (and the AbstractSession import) can be removed since it is now identical to the `RestClient` constructor.